nebula-logger/core/main/configuration/customMetadata/LoggerParameter.StoreApexHeapSizeLimit.md-meta.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomMetadata
+    xmlns="http://soap.sforce.com/2006/04/metadata"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+>
+    <label>Store Apex Heap Size Limits</label>
```

Comment:
Same type of thing here, the label is using plural "limits" but the dev name is singular 'limit"